deckhouse-controller/pkg/helpers/register.go

```diff
@@ -35,6 +36,13 @@ func DefineHelperCommands(kpApp *kingpin.Application) {
 		})
 	}
 
+	{
+		collectDebug := helpersCommand.Command("collect-debug-info", "Collect debug info from your cluster.")
```

Comment:
Is it a "helper"? As I rememeber, helpers are handy commands for shell hooks. I think `deckhouse-controller collect-debug-info` is ok.
```suggestion
		collectDebug := sh_app.CommandWithDefaultUsageTemplate(kpApp, "collect-debug-info", "Collect debug info from your cluster.")
```